code/modules/admin/verbs/listahelps.dm

```diff
@@ -0,0 +1,307 @@
+/datum/adminticket
+	var/ID = "" //ID of the ticket, very important as its used to find adminhelps.
+	var/user = "" //The user of the ahelp.
+	var/uckey //The saved ckey of the adminheloing user.
+	var/admin = "N/A" //The handling admin? Like come on.
+	var/msg = "" //The adminhelp message.
+	var/resolved = "No" //Is it resolved? Its much easier to have a "Yes" or a "No", as you can directly concat it into strings making life that much easier.
```

Comment:
Mmm. Halfway into implimenting, I realized something.

You do know that the strings allow you do see if it's resolved or not via the chat scrren, yeah?

https://i.gyazo.com/e7d6bff18d3cf88936f84cfe36f5162a.png aka this.